eos-live-boot-dm-setup.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Endless live boot device mapper setup
```

Comment:
Both in the filenames in the descriptions, can you rename this to something without "live", since it is used for both live boots (meaning the root device is kept read only and we put on a lot of overlayfs on top) and non-live boots (e.g. dual boot with windows). Perhaps eos-image-boot-dm-setup or something like that